k8s/deploy_app.sh

```diff
@@ -24,7 +24,8 @@ helm upgrade --install \
     --set-string autoscaler.targetCPUUtilizationPercentage="${TARGET_CPU_UTILIZATION_PERCENTAGE}" \
     --set-string newRelic.enabled="${EQ_NEW_RELIC_ENABLED}" \
     --set-string newRelic.licenseKey="${NEW_RELIC_LICENSE_KEY}" \
-    --set-string newRelic.appName="${NEW_RELIC_APP_NAME}"
+    --set-string newRelic.appName="${NEW_RELIC_APP_NAME}" \
+    --set-string individualResponseLimit="${EQ_INDIVIDUAL_RESPONSE_LIMIT}"
```

Comment:
This should be removed as it will break any deployments that don't set `EQ_INDIVIDUAL_RESPONSE_LIMIT`, as seen with `CONFIRMATION_EMAIL_LIMIT` recently in https://github.com/ONSdigital/eq-questionnaire-runner/pull/330.
```suggestion
    --set-string newRelic.appName="${NEW_RELIC_APP_NAME}"
```

We need to decide how we do this so that the default will be used if the environment variable isn't set.